internal/services/redis/redis_cache_resource_test.go

```diff
@@ -1291,7 +1383,11 @@ resource "azurerm_redis_cache" "test" {
 func (RedisCacheResource) userAssignedIdentity(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
this would indicate something isn't being deleted correctly which would cause issues for real users - rather than adding this workaround, can we fix the issue here?